src/router.js

```diff
@@ -17,8 +17,14 @@ export default function (options) {
       }
     },
     hooks: {
+      onUpdate: function (oldModel, newModel, data, actions) {
+        if(newModel.router && oldModel.router.path !== newModel.router.path &&
```

Comment:
It relies on `oldModel.router.path !== newModel.router.path` where `path` is  https://github.com/hyperapp/hyperapp/pull/150/files#diff-15ee858ae5d5aa9b3e2846cfd1c4c466R71

The `newModel.router` check is there to prevent an error being thrown when trying to access `newModel.router.path` when `newModel.router` does not exist. Also quite a cheap way of checking if any action needs to be considered at all.